ledger/src/shred/merkle.rs

```diff
@@ -193,6 +193,18 @@ impl ShredData {
         Ok(Self::SIZE_OF_HEADERS + Self::capacity(proof_size, /*chained:*/ true, resigned)?)
     }
 
+    fn get_chained_merkle_root_offset(proof_size: u8, resigned: bool) -> Result<usize, Error> {
```

Comment:
if adding this function, then `fn chained_merkle_root_offset` should reuse it as opposed to duplicating that last line.